src/loaders.py

```diff
@@ -100,109 +101,8 @@ def __getitem__(self, index):
-
-        self.train_mode = train_mode
-        self.image_transform = image_transform
-        self.mask_transform = mask_transform
-        self.image_augment = image_augment
-        self.image_augment_with_target = image_augment_with_target
-
-    def load_image(self, img_filepath):
-        image = Image.open(img_filepath, 'r')
-        return image.convert('RGB')
-
-    def __len__(self):
-        return self.X.shape[0]
-
-    def __getitem__(self, index):
-        img_filepath = self.X[index]
-
-        Xi = self.load_image(img_filepath)
-        if self.y is not None:
-            target_masks = []
-            for i in range(y.shape[1]):
-                filepath = self.y[index, i]
-                mask = self.load_image(filepath)
-                target_masks.append(mask)
-            target_masks = [target[index] for target in self.y]
-            data = [Xi] + target_masks
-
-            if self.train_mode and self.image_augment_with_target is not None:
-                data = from_pil(*data)
-                data = self.image_augment_with_target(*data)
-                data[0] = self.image_augment(data[0])
-                data = to_pil(*data)
-
-            if self.mask_transform is not None:
-                data[1:] = [self.mask_transform(mask) for mask in data[1:]]
-
-            if self.image_transform is not None:
-                data[0] = self.image_transform(data[0])
-
-            return data
-        else:
-            if self.image_transform is not None:
-                Xi = self.image_transform(Xi)
-            return Xi
-
-
-class ImageSegmentationMultitaskDataset(Dataset):
-    def __init__(self, X, y, train_mode,
-                 image_transform, image_augment_with_target,
-                 mask_transform, image_augment):
-        super().__init__()
-        self.X = X
-        if y is not None:
-            self.y = y
-        else:
-            self.y = None
 
-        self.train_mode = train_mode
-        self.image_transform = image_transform
-        self.mask_transform = mask_transform
-        self.image_augment = image_augment
-        self.image_augment_with_target = image_augment_with_target
-
-    def __len__(self):
-        return len(self.X[0])
-
-    def __getitem__(self, index):
-        Xi = self.X[0][index]
-
-        if self.y is not None:
-            target_masks = [target[index] for target in self.y]
-            data = [Xi] + target_masks
-
-            if self.train_mode and self.image_augment_with_target is not None:
-                data = from_pil(*data)
-                data = self.image_augment_with_target(*data)
-                data[0] = self.image_augment(data[0])
-                data = to_pil(*data)
-
-            if self.mask_transform is not None:
-                data[1:] = [self.mask_transform(mask) for mask in data[1:]]
-
-            if self.image_transform is not None:
-                data[0] = self.image_transform(data[0])
-
-            return data
```

Comment:
OK, so now we remove the multi-output UNet, right? @jakubczakon are you aware of that?

Reply:
@apyskir I think we were talking about this, but waiting for solution 2 implementation with that. I could be wrong though.

Reply:
@taraspiotr @apyskir yes I am aware, remember that we are reimplementing the winning solution. Our solution still has this transformer. 

I will move those loader transformers to `steppy-toolkit` in the following weeks so we are not going to lose any of that.
@apyskir thanks for asking!

Reply:
@jakubczakon @taraspiotr OK, got it.